packages/salesforcedx-vscode-apex/test/views/readableApexTestRunExecutor.test.ts

```diff
@@ -0,0 +1,83 @@
+/*
```

Comment:
This file adds tests to check the commands being generated by `readableApexTestRunExecutor` to run the apex tests through the sidebar.